src/assessment.js

```diff
@@ -27,7 +27,7 @@ var q1 = /* your answer here */;
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
+var q2 = [/* 'alderaan', 'deathStar', 'fire' */];
```

Comment:
Only the `deathStar` and `fire` scopes have access to `laserQuantity`. This is because `laserQuantity` is defined within `deathStar`, so it isn't accessible outside of it (function scope), and the `fire` function is also defined within `deathStar`, so everything _inside_ `fire` also has access.